### PR TITLE
Checkbox styles

### LINKED ISSUE #499: Theme Update - CheckBox Style
[Design](https://www.figma.com/file/K0PppsSh6aQSszqlwKAekg/hpe-design-system-stickers?node-id=425%3A0)

CheckBox should be:
- [x]  24x24
- [x] border-radius: 2px
- [x] border-width: 1px (borderSize 'xsmall')
- background-color, selected: 'selected-background'
- [x] checkmark color 'selected-text'
- use new checkmark svg
- [x] Spacing in between checkbox and checkbox label should be gap 'small' (12px. --- stated goal is this spacing should be the same as between button text and button icon)

### PR DESCRIPTION
**Pull request by britt6612**
Related Issues: #499

closes #499

[Preview](https://deploy-preview-543--keen-mayer-a86c8b.netlify.com/templates/forms)
[Figma](https://www.figma.com/file/K0PppsSh6aQSszqlwKAekg/hpe-design-system-stickers?node-id=425%3A0)

Updated the theme for the checkbox including the following 

- [x]  24x24
- [x] border-radius: 2px
- [x] border-width: 1px (borderSize 'xsmall')
- [x] checkmark color 'selected-text'
- [x] Spacing in between checkbox and checkbox label should be gap 'small' (12px. --- stated goal is this spacing should be the same as between button text and button icon)

One small thing I noticed is the checkmark looks bold in the preview vs designs so not sure if the new svg was going to help with that or if we should look at not using selected-text which is the bold

Work todo 
issue #544

### CODE REVIEW

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -20,11 +20,19 @@ export const aries = deepMerge(hpe, {
...
   },
   checkBox: {
+    color: 'selected-text',
+    gap: 'small',
     toggle: {
       color: {
         dark: hpe.global.colors['text-strong'].dark,
         light: hpe.global.colors['text-strong'].dark,
       },
+      check: {
```
Comment (taysea) [inline aries-site/src/themes/aries.js]: I think these changes should be happening just under checkBox as opposed to inside toggle

Comment (britt6612) [reply aries-site/src/themes/aries.js]: good catch I didnt mean to have these in toggle

File: aries-site/src/themes/aries.js [hunk 2]
```diff
@@ -6,20 +6,23 @@ export const aries = deepMerge(hpe, {
   global: {
     colors: {
       focus: 'teal!',
-      placeholder: 'text-xweak',
```
Comment (taysea) [inline aries-site/src/themes/aries.js]: You might not be up to date with master here, but I don't think we want to remove this block.

File: aries-site/src/themes/aries.js [hunk 3]
```diff
@@ -6,20 +6,23 @@ export const aries = deepMerge(hpe, {
...
     },
     input: {
       weight: 500,
     },
   },
-  button: {
```
Comment (taysea) [inline aries-site/src/themes/aries.js]: Same as above, don't think we want to remove this.

File: aries-site/src/themes/aries.js [hunk 4]
```diff
@@ -6,20 +6,23 @@ export const aries = deepMerge(hpe, {
...
-    color: props =>
-      props.primary ? '#FFFFFF' : normalizeColor('text', props.theme),
-  },
   layer: {
     background: 'background',
   },
   checkBox: {
+    color: 'selected-text',
+    gap: 'small',
+    check: {
```
Comment (taysea) [inline aries-site/src/themes/aries.js]: There is a theme attribute of `checkBox.check.radius` that we can use here instead of extend.

Comment (britt6612) [reply aries-site/src/themes/aries.js]: I missed that thanks

**Pull request merged**